backend/app/services/executor_streaming/state.py

```
@@ -0,0 +1,445 @@
+            chunk: Content chunk to append
+            expire: TTL in seconds
+
+        Returns:
+            Tuple of (success, new_total_length)
+        """
+        try:
+            existing = await self.get_streaming_content(subtask_id) or ""
+            new_content = existing + chunk
+            success = await self.save_streaming_content(subtask_id, new_content, expire)
+            return success, len(new_content)
+        except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)